core/tests/consensus.rs

```diff
@@ -137,7 +139,8 @@ fn get_diff_stats(chain_sim: &Vec<Result<(u64, Difficulty), TargetError>>) -> Di
 
 	// Obtain the median window for the latest time period
 	// i.e. the last  MEDIAN_TIME_WINDOW elements
-	let mut window_latest: Vec<u64> = last_blocks.clone()
```

Comment:
But then here it appears to be the opposite... (was on same line, now on next line).

Reply:
:+1: for a whole-project rustfmt commit soon-ish. Then if rustfmt issues continue to pop up, maybe check tab size configs and maybe use .editorconfig in project root